udata_metrics/client.py

```diff
@@ -1,26 +1,104 @@
+import logging
+from datetime import datetime, timedelta
+import uuid
 from flask import current_app
-from influxdb import InfluxDBClient
+from influxdb_client import InfluxDBClient
 
+from udata.core.dataset.models import Dataset, get_resource
+from udata.models import Reuse, Organization
+
+log = logging.getLogger(__name__)
 
 class InfluxClient:
 
     def __init__(self, dsn):
-        self.client = InfluxDBClient(**dsn)
+        self.client = InfluxDBClient(**dsn, timeout=600_000)
+
+    def aggregate_metrics(self, measurement: str):
```

Comment:
Could we maybe split this function in multiple functions, since there are 3 different main parts?